src/platformio.ini

```diff
@@ -58,8 +58,6 @@ build_flags =
 	-D PLATFORM_STM32=1
 	-O2
 src_filter = ${common_env_data.src_filter} -<ESP32*.*> -<ESP8266*.*> -<WS281B*.*>
-lib_deps =
```

Comment:
revert these changes. you can just define empty `lib_deps =` if you want to remove this.